crates/router/src/types/api/payments.rs

```diff
@@ -9,7 +9,8 @@ pub use api_models::payments::{
     PaymentsApproveRequest, PaymentsCancelRequest, PaymentsCaptureRequest,
     PaymentsCompleteAuthorizeRequest, PaymentsDynamicTaxCalculationRequest,
     PaymentsDynamicTaxCalculationResponse, PaymentsExternalAuthenticationRequest,
-    PaymentsIncrementalAuthorizationRequest, PaymentsManualUpdateRequest, PaymentsRedirectRequest,
+    PaymentsIncrementalAuthorizationRequest, PaymentsManualUpdateRequest,
```

Comment:
@SanchithHegde what do you think about such re exports? This adds friction when a developer is trying to move his way to from where and how this struct has been imported. In my opinion we should always qualify specific imports from the parent module. This might take some time, but I think it will be better to move away from this.

Reply:
I believe this was kept this way when we moved out API models to a separate crate, to reduce the number of changes elsewhere in code. But yeah, I think we should avoid such public re-exports.

Of course, this can be taken up in a separate PR.